server/game/cards/07.3-BadM/MotherBearsRage.js

```diff
@@ -0,0 +1,41 @@
+const GameActions = require('../../GameActions/index.js');
+const SpellCard = require('../../spellcard.js');
+
+class MotherBearsRage extends SpellCard {
+    setupCardAbilities(ability) {
+        this.spellAction({
+            title: 'Noon',
+            playType: 'noon',
+            cost: ability.costs.bootSelf(),
+            target: {
+                activePromptTitle: 'Select a dude to call out',
+                waitingPromptTitle: 'Waiting for opponent to select a dude',
+                cardCondition: { condition: card => (card.gamelocation === this.gamelocation ||
+                    card.isAdjacent(this.gamelocation) && !card.isInTownSquare())},
```

Comment:
```suggestion
                cardCondition: { controller: 'opponent', condition: card => (card.gamelocation === this.gamelocation ||
                    card.isAdjacent(this.gamelocation) && card.locationCard.getType() === 'deed')},
```
This allows to call out also own dudes and dudes at adjacent outfit, but only adjacent deeds are allowed.